packages/features/eventtypes/components/CheckedTeamSelect.tsx

```diff
@@ -4,6 +4,7 @@ import type { Dispatch, SetStateAction } from "react";
 import { useFormContext } from "react-hook-form";
 import type { Props } from "react-select";
 
+import type { Host, FormValues } from "@calcom/features/eventtypes/lib/types";
```

Comment:
type-check if passing now with this import, that was a tricky one

Reply:
@CarinaWolli Indeed a tricky one, thanks for taking the time to fixing it :)